cronjobs.yml

```diff
@@ -15,7 +15,15 @@ cronJobs:
         memory: "5Mi"
       limits: # maximum allowed resources
         cpu: "1.5"
-        memory: "600Mi"
+        memory: "1.5Gi"
```

Comment:
The process-test-mapping job is terminating due to an OOMKilled error so I'm increasing to the same limit we put on the process-task-mapping which is succeeding
![Screen Shot 2020-03-09 at 3 19 10 PM](https://user-images.githubusercontent.com/7087489/76252940-16402700-6220-11ea-9188-4862d2cf4119.png)